### ISSUE TITLE
Hangfire table date columns save date in UTC time, can it be changed to save the CST time instead?

### ISSUE THREAD

**Issue by prichakraborty09** [HangfireIO/Hangfire#1898]
Hi,

When I query from the tables HangFire.Job, HangFire.State, I have observed that the date columns save the UTC time. Is it possible to save the dates as CST time instead ?

Thanks
PC

**Comment by prichakraborty09** [comment]
@Frazi1 any idea on this one ?

**Comment by Frazi1** [comment]
I don't think you can change the stored time.

However, if I'm not mistaken, some Hangfire features can operate in different timezones. For example, it should be possible to configure recurring jobs for a specific timezone.

What are you trying to accomplish by switching to CST?

**Comment by prichakraborty09** [comment]
@Frazi1 you're right. So I have configured a recurring job in CST timezone in my Startup.cs file. When I refer the tables, instead of storing the various dates in CST time, it is saved in UTC format. I would like it to be CST so that it is easier for our team to debug and analyse whenever required because our job is required to execute every CST hour.

**Comment by pieceofsummer** [comment]
Storing dates in local timezone may result in missed/duplicate runs when daylight saving change occurs, so it generally a bad idea. If you're operating data on database level you can create views with date columns converted to a local timezone for better readability.

**Comment by prichakraborty09** [comment]
@pieceofsummer The requirement was so because all our date gets saved in CST format, so thought that if Hangfire tables also could store date in CST format it becomes uniform.
